google/resource_endpoints_service.go

```diff
@@ -111,6 +124,20 @@ func resourceEndpointsService() *schema.Resource {
 	}
 }
 
+func migrateEndpointsService(v int, is *terraform.InstanceState, meta interface{}) (*terraform.InstanceState, error) {
+	if v == 0 {
```

Comment:
we usually use switch/case statements for this, which are nice since they let you fall through (so if a user is on v0 and the latest one is v2, we can sequentially migrate 0->1 and then 1->2). doesn't quite matter now, but will make it easier to extend later if need be

Reply:
Done.